cmd/kpod/ps.go

```diff
@@ -135,12 +135,16 @@ func psCmd(c *cli.Context) error {
 	if c.IsSet("format") {
 		format = c.String("format")
 	}
+	last := -1
+	if c.IsSet("last") {
+		last = c.Int("last")
+	}
 
 	opts := psOptions{
 		all:     c.Bool("all"),
 		filter:  c.String("filter"),
 		format:  format,
-		last:    c.Int("last"),
+		last:    last,
```

Comment:
last: c.Int("last")

Reply:
Right, I keep forgetting we can set default values. Fixed.